modules/installation-and-upgrade/pages/network-requirements.adoc

```diff
@@ -37,6 +28,28 @@ You also need to ensure that reverse lookups are correctly configured.
 
 For more information about setting up a DNS server, see {sles-base-os-documentation}/cha-dns.html.
 
+
+== Reenable router advertisements
+
+When the {productname} is installed using [command]``mgradm podman install``, it sets up Podman which enables IPv4 and IPv6 forwarding.
+This is needed for communication from the outside of the container.
+
+However, if your system previously had [literal]``/proc/sys/net/ipv6/conf/eth0/accept_ra`` set to [literal]``1``, it will stop using router announcements.
+As a result, the routes are no longer obtained via router advertisements and the default IPv7 route is missing.
+
+To enable correct functioning of the IPv6 routing, follow the procedure:`
+
+.Reenabling router advertisments
+. Create a file in [path]``/etc/sysctl.d``, for example [literal]``99-ipv6-routing.conf``.
```

Comment:
maybe 99-ipv6-ras.conf would be a better name

(i'm not even completely sure, in fact)

Reply:
This is a suggested name.